cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -247,7 +258,7 @@ func Validate(args *Args, cfg *config.Config) error {
 	cfg.EnableNydusOverlayFS = args.EnableNydusOverlayFS
 	cfg.NydusdThreadNum = args.NydusdThreadNum
 	cfg.CleanupOnClose = args.CleanupOnClose
-	cfg.DaemonBackend = args.DaemonBackend
+	cfg.DaemonBackend = args.FsDriver
```

Comment:
`cfg.FsDriver`? Maybe we need to rename all `DaemonBackend`.

Reply:
Yes. Already renamed them all